autogen/token_count_utils.py

```diff
@@ -158,7 +146,7 @@ def _num_token_from_messages(messages: Union[List, Dict], model="gpt-3.5-turbo-0
             num_tokens += len(encoding.encode(value))
             if key == "name":
                 num_tokens += tokens_per_name
-    num_tokens += 3  # every reply is primed with <|start|>assistant<|message|>
+    num_tokens += 2  # every reply is primed with <im_start>assistant
```

Comment:
Is "num_tokens" changed for all OpenAI models, or just the O1 model? 

If it is just for the O1 model, we should adjust this "if statement" above.

Reply:
I looked through the advice for counting and these are the numbers covering all models (perhaps it has changed over time), so I updated it universally.